modules/apps/object/object-dynamic-data-mapping-form-field-type/src/main/java/com/liferay/object/dynamic/data/mapping/form/field/type/internal/attachment/AttachmentDDMFormFieldTemplateContextContributor.java

```diff
@@ -154,13 +154,6 @@ private Map<String, String> _getFileEntryProperties(
 	private long _getGroupId(
 		DDMFormFieldRenderingContext ddmFormFieldRenderingContext) {
 
-		long groupId = GetterUtil.getLong(
-			ddmFormFieldRenderingContext.getProperty("groupId"));
-
-		if (groupId != 0) {
-			return groupId;
-		}
```

Comment:
I don't think we can remove this part of code because we have Objects scoped by Site.
You would have to compare the value returned by `ddmFormFieldRenderingContext.getProperty("groupId")` to the value returned `themeDisplay.getCompanyGroupId()`.

I think you need to test if your solution will impact the Objects scoped by Site to not cause regression bugs. Do you agree? :smile: